### ISSUE TITLE
[Android] AutomatedTesting.GameLauncher displays a black screen and does not load levels

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de#11768]
**Describe the bug**
Launching the AutomatedTesting.GameLauncher deployed to a device built for Android displays a black screen and does not further load any levels.

Please refer to the attached video and logs for more information.

**Prerequisites**
- AutomatedTesting project is built for Windows.
- Assets for the Android platform are enabled and processed.
- An autoexec.cfg file has been created inside the _/o3de/AutomatedTesting_ directory containing:
`LoadLevel levels/graphics/ShadowTest/ShadowTest.spawnable`
- All the required environment variables are set as described in the instructions: https://o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/

In this test the following settings have been used:

```
SET O3DE_ENGINE_PATH=C:\o3de
SET O3DE_PROJECT_PATH=C:\o3de\AutomatedTesting
SET O3DE_BUILD_ROOT=C:\o3de-build
```
```
SET O3DE_ANDROID_SIGNCONFIG_FILE=%O3DE_BUILD_ROOT%\o3de.keystore
SET O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD=o3de_android
SET O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS=o3de_key
SET O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD=o3de_android
SET O3DE_ANDROID_SIGNCONFIG_KEY_SIZE=2048
SET O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS=10000
SET O3DE_ANDROID_DN="cn=AutomatedTesting, ou=o3de, o=LF, c=US"
```
```
set O3DE_ANDROID_SDK_PATH=C:\Android\Sdk
set O3DE_ANDROID_NDK_VERSION=22.1.7171670
set O3DE_ANDROID_SDK_API_LEVEL=30
set O3DE_ANDROID_ASSET_MODE=LOOSE
set O3DE_ANDROID_DEPLOY_TYPE=BOTH
```

**Steps to reproduce**
1. Generate KeyStore to allow APK signing:
`keytool -genkey -keystore %O3DE_ANDROID_SIGNCONFIG_FILE% -storepass %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% -alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% -keypass %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% -keyalg RSA -keysize %O3DE_ANDROID_SIGNCONFIG_KEY_SIZE% -validity %O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS% -dname %O3DE_ANDROID_DN%`
2. Use Python to run generate_android_project.py:
`%O3DE_ENGINE_PATH%\python\python.cmd %O3DE_ENGINE_PATH%\cmake\Tools\Platform\Android\generate_android_project.py --engine-root %O3DE_ENGINE_PATH% --project-path %O3DE_PROJECT_PATH% --build-dir %O3DE_BUILD_ROOT%\android --third-party-path %USERPROFILE%\.o3de\3rdParty --android-sdk-path %O3DE_ANDROID_SDK_PATH% --android-ndk-version %O3DE_ANDROID_NDK_VERSION% --android-sdk-platform %O3DE_ANDROID_SDK_API_LEVEL% --include-apk-assets --asset-mode %O3DE_ANDROID_ASSET_MODE% --signconfig-store-file %O3DE_ANDROID_SIGNCONFIG_FILE% --signconfig-store-password %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% --signconfig-key-alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% --signconfig-key-password %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD%`
3. Build the Android project:
`cd %O3DE_BUILD_ROOT%\android`
`gradlew assembleProfile`
4. Deploy the GameLauncher to the Android device:
`%O3DE_ENGINE_PATH%\python\python.cmd %O3DE_ENGINE_PATH%\cmake\Tools\Platform\Android\deploy_android.py --build-dir %O3DE_BUILD_ROOT%\android --configuration profile --clean -t %O3DE_ANDROID_DEPLOY_TYPE%`
5. Launch the deployed GameLauncher.

**Expected behavior**
The deployed GameLauncher launches without issues.

**Actual behavior**
The deployed GameLauncher displays black screen and does not load any levels.

**Video**

https://user-images.githubusercontent.com/86953659/188430890-24dd08ca-7032-4687-b5e6-1a431baff11a.mp4

**Found in Branch**
Stabilization/2210 (99608ff)

**Device**
 - Device: Samsung S20+
 - OS: Android
 - Version 11 (API 30)

**Additional context**
[adb_logcat_AutomatedTesting.txt](https://github.com/o3de/o3de/files/9489183/adb_logcat_AutomatedTesting.txt)

**Comment by lumberyard-employee-dm** [comment]
Should be fixed by PR #11781

**Comment by LB-KacperKapusta** [comment]
Verified the issue as fixed using the Stabilization/2210 (58fb16c) build.
